packages/flutter/lib/src/gestures/pointer_router.dart

```diff
@@ -108,6 +108,9 @@ class PointerRouter {
   }
 }
 
+// TODO(jacobr): we could alternately remove this library as these properties
+// could just as well be injected into the FlutterError object if they are
+// useful.
 /// Variant of [FlutterErrorDetails] with extra fields for the gestures
```

Comment:
If there's a way to remove FlutterErrorDetailsForPointerRouter and have the code be cleaner in the process, I'm all for it. The existence of this class is unfortunate IMHO.

Reply:
Yes. I would just generate FlutterError objects describing these properties. There are only a couple cases that use this class so it doesn't pay its weight.
The diagnostics could either have a low priority so they only show up in tools or a high priority so everyone sees these properties as they seem quite useful.